app/views/finders/_facet_collection.html.erb

```diff
@@ -1,4 +1,11 @@
 <div class="filter-form">
+  <% if content_item.all_content_finder? %>
```

Comment:
Is this `if` block over-complicated? Could it be written instead as

```Ruby
<% if content_item.all_content_finder? or facets.select(&:filterable?).any? %>
  <%= render partial: 'filter_button'%>
<% end %>
```

Reply:
No worries, yeah I didn't change any Ruby logic in this PR luckily! If it was on one line, wouldn't it need to be:

```Ruby
<% if content_item.all_content_finder? or !content_item.all_content_finder? and facets.select(&:filterable?).any? %>
  <%= render partial: 'filter_button'%>
<% end %>
```

as in the original, the `elsif` only runs if `content_item.all_content_finder?` is `false`, whereas that simplified solution wouldn't check for that.

Reply:
Yes, I think my solution was a little too simple, but isn't the elsif unnecessary? The code is basically saying

```
if a_thing
  (render partial)
elsif not a_thing
  if another_thing
    (render partial)
  end
end
```

i.e. there are only two possible outcomes for the top level if statement - a_thing is either or not, so it's always going to run one of those statements. Doesn't that mean it could be

```
if a_thing
  (render partial)
elsif another_thing
  (render partial)
end
```

I concede that I still might be missing something, it's Friday afternoon, so happy for you to point out the flaw in my logic.

Reply:
@andysellick That example makes sense - I think because your original suggestion uses `or` it would've changed the flow to:

```
    if a_thing or another_thing
        (render partial)
```

instead of 

```
    if a_thing
      (render partial)
    elsif another_thing
      (render partial)
    end
```

and no worries, I have Friday brain too

Reply:
Thanks for discussing this with me just now Ash. As discussed, I think we both understand this a bit more clearly but on the basis that this is a bit of code that's been moved rather than written as part of this PR, I think we should leave it as it is for now.